projects/storefrontlib/src/cms-components/organization/units/index.ts

```diff
@@ -12,3 +12,18 @@ export * from './unit-address-edit/unit-address-edit.module';
 export * from './unit-address-details/unit-address-details.module';
 export * from './unit-children/unit-children.module';
 export * from './unit-cost-centers/unit-cost-centers.module';
+
+export * from './unit-details/unit-details.component';
+export * from './unit-create/unit-create.component';
+export * from './unit-edit/unit-edit.component';
+export * from './unit-list/unit-list.component';
+export * from './unit-users/unit-users.component';
+export * from './unit-assign-roles/unit-assign-roles.component';
+export * from './unit-assign-approvers/unit-assign-approvers.component';
+export * from './unit-approvers/unit-approvers.component';
+export * from './unit-manage-addresses/unit-manage-addresses.component';
+export * from './unit-address-create/unit-address-create.component';
+export * from './unit-address-edit/unit-address-edit.component';
+export * from './unit-address-details/unit-address-details.component';
+export * from './unit-children/unit-children.component';
+export * from './unit-cost-centers/unit-cost-centers.component';
```

Comment:
why do you export components? we export modules for that

Reply:
Was just an experiment to tackle the prod storefrontlib build issue, already solved and brought back.